fe/fe-core/src/main/java/org/apache/doris/catalog/SparkResource.java

```diff
@@ -159,16 +160,17 @@ public Map<String, String> getSparkConfigs() {
     }
 
     public Map<String, String> getEnvConfigsWithoutPrefix() {
-        Map<String, String> envConfig = Maps.newHashMap();
+        Map<String, String> systemEnv = System.getenv();
+        Map<String, String> currentEnvConfigs = new HashMap<>(systemEnv);
```

Comment:
This line includes all system environment variables. What about including ones with `ENV_PREFIX` prefixes?